packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -30,18 +61,24 @@ export const useVisibilityTreeRenderer = (iconsEnabled: boolean, descriptionsEna
 };
 
 const imageLoader = new TreeImageLoader();
+
 /**
  * Creates node renderer which renders node with eye checkbox.
  * @public
  */
-export const createVisibilityTreeNodeRenderer = (iconsEnabled: boolean, descriptionEnabled: boolean) => {
-  return (props: TreeNodeRendererProps) => ( // eslint-disable-line react/display-name
+export const createVisibilityTreeNodeRenderer = ({ levelOffset = 20, expansionToggleWidth = 24, disableRootNodeCollapse = false, ...props }: visibilityTreeRendererProps) => {
+  return (treeNodeProps: TreeNodeRendererProps) => ( // eslint-disable-line react/display-name
     <TreeNodeRenderer
-      {...props}
-      checkboxRenderer={visibilityTreeNodeCheckboxRenderer}
-      descriptionEnabled={descriptionEnabled}
-      imageLoader={iconsEnabled ? imageLoader : undefined}
-      className={classNames("with-checkbox", props.className)}
+      {...treeNodeProps}
+      node={{ ...treeNodeProps.node, depth: 0, numChildren: 1 }}
+      checkboxRenderer={(checkboxProps: NodeCheckboxRenderProps) => (
+        <div className="checkboxWrapper" style={{ marginRight: `${treeNodeProps.node.depth * levelOffset + (treeNodeProps.node.numChildren === 0 ? expansionToggleWidth : 0)}px` }}>
```

Comment:
Can you move margin calculations into some variable instead of inlining in style object?

Reply:
fixed